app/src/components/menu.ts

```diff
@@ -1,4 +1,6 @@
 import { Menu, clipboard, shell, MenuItemConstructorOptions } from 'electron';
+import * as fs from 'fs';
+import path from 'path';
```

Comment:
Nit about imports ordering: 1. node stdlib, 2. deps, 3. internal, separated by newlines. So,

```suggestion
import * as fs from 'fs';
import path from 'path';

import { Menu, clipboard, shell, MenuItemConstructorOptions } from 'electron';
```